packages/hop-node/src/watchers/SyncWatcher.ts

```diff
@@ -1686,6 +1742,53 @@ class SyncWatcher extends BaseWatcher {
       return true
     }
   }
+
+  // Experimental: Websocket support methods
+
+  initWebsocket = (contract: Contract, filter: EventFilter, cb: Function) => {
```

Comment:
I don't think this needs to be an arrow function but if it does need to be one, then maybe adding a comment about it be good here

Reply:
Good call. Resolved with d6fbb6dc38f2876435995b243b1d2ac2218c2554